WordPressFlux/WordPressFlux/Sources/QueryStore.swift

```diff
@@ -17,23 +17,27 @@ private struct QueryRef<Query> {
 /// Subclasses need to implement queriesChanged() and look at activeQueries and
 /// their internal state to decide if they need to act.
 ///
-/// If your `State` type confrorms to the `CachableState` protocol, the `QueryStore`
-/// will transparently persist/load cached State and purge in-memory State when there are no
+/// If your `State` type confrorms to the `Codable` protocol, the `QueryStore`
+/// will transparently persist/load State and purge in-memory State when there are no
 /// longer any active `Queries` associated with the `Store`.
 ////
-/// You can also manually force writing of the cache file by calling `storeCachedState()`.
+/// You can also manually force writing out the state to disk by calling `persistState()`.
 ///
 
 open class QueryStore<State, Query>: StatefulStore<State>, Unsubscribable {
     private let initialState: State
 
     fileprivate var activeQueryReferences = [QueryRef<Query>]() {
         didSet {
-            if activeQueryReferences.isEmpty, let encodableState = state as? Encodable, let url = try? type(of: self).cacheFileURL() {
+            if activeQueryReferences.isEmpty, let encodableState = state as? Encodable, let url = try? type(of: self).persistenceURL() {
                 // If we don't have any active queries, and the `state` conforms to `Encodable`, let's use this as our cue to persist the data
                 // to disk and get rid of the in-memory cache.
-                inMemoryState = nil
-                encodableState.storeCachedState(at: url)
+                do {
+                try encodableState.saveJSON(at: url)
```

Comment:
Nitpick but I think this line needs another tab 😄

Reply:
huh, weird that the linter didn't pick it up!